mysql-to-kv-delta/mysql-to-kv-delta.py

```diff
@@ -0,0 +1,59 @@
+
+
+      
+
+# Copyright 2017 The Nuclio Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import pandas as pd
+import time
+import datetime
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+import v3io_frames as v3f
+
+
+def handler(context, event):
+    delta_interval_minute = os.getenv('DELTA_INTERVAL_MINUTE')
+    modified_col = os.getenv('MODIFIED_DATETIME_COL')
+    created_col = os.getenv('CREATED_DATETIME_COL')
+    datetime_query = str(datetime.datetime.now() - datetime.timedelta(seconds=int(delta_interval_minute)*60))
+    sql_query = os.getenv('SQL_QUERY')
+    sql_query_diff = sql_query + " where ("+created_col+">='"+str(datetime_query)+"' AND "+modified_col+" IS NULL) OR ("+modified_col+">='"+str(datetime_query)+"')"
+    context.logger.debug(sql_query_diff)
```

Comment:
is this logging necessary? (this comment is relevant to all the other log messages in this PR)
if so, then make it readable by adding a relevant text before it explaining what this log is about

Reply:
the logging is only in debug mode.

Reply:
ok 👍 
please just add a matching message describing the value you are logging, something like:
```
context.logger.debug('Generated sql query', sql_query_diff=sql_query_diff)
```